PulleyLib/PulleyViewController.swift

```diff
@@ -1606,20 +1599,42 @@ extension PulleyViewController: UIScrollViewDelegate {
                         }
                     }
                 }
-                
-                setDrawerPosition(position: positionToSnapTo, animated: true)
+
+                closestValidDrawerPosition = positionToSnapTo
             }
-        }
-    }
-    
-    public func scrollViewWillEndDragging(_ scrollView: UIScrollView, withVelocity velocity: CGPoint, targetContentOffset: UnsafeMutablePointer<CGPoint>) {
-        
-        if scrollView == drawerScrollView
-        {
-            lastDragTargetContentOffset = targetContentOffset.pointee
-            
-            // Halt intertia
-            targetContentOffset.pointee = scrollView.contentOffset
+
+            self.drawerPosition = closestValidDrawerPosition
+
+            let stopToMoveTo: CGFloat
```

Comment:
Note for enhancement: we could extract that line and the switch to a separate method